src/ui/package.json

```diff
@@ -40,7 +40,7 @@
     "objectpath": "^2.0.0",
     "startbootstrap-sb-admin-2": "^3.3.7",
     "swagger-ui-dist": "^3.25.0",
-    "tv4": "~1.0.15",
+    "tv4": "~1.3.0",
```

Comment:
What drove this update?

Reply:
The LTS version of *npm* is 6.14.15. Using that version of *npm*, the library at  version 1.0.15 throws up a number of warnings that don't exist at a newer version. The author doesn't appear to publish a change log on his github, but I'm assuming a minor version bump will not break its API.